dune-project

```diff
@@ -0,0 +1,48 @@
+(lang dune 2.7)
+(name extunix)
+(generate_opam_files true)
+
+(package
+ (name extunix)
+ (depends
+  (dune-configurator :build)
+  ppxlib
```

Comment:
This most likely need a lower-bound version constraint, `>= "0.18.0"` at least, i suspect, due to the use of `Pconst_string` with 3 arguments.

Reply:
I want to use OCaml 4.12, so I'll go with 0.22.